specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkManagerActiveConfiguration.json

```
@@ -0,0 +1,171 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagerActiveConfiguration",
+    "description": "The Microsoft Azure Network Manager Activity Configuration API provides a RESTful set of web services that interact with Microsoft Azure Network Manager service to manage your network resources.",
```

Is "Activity"  the right word here, or just "Active" .

```suggestion
    "description": "The Microsoft Azure Network Manager Active Configuration API provides a RESTful set of web services that interact with Microsoft Azure Network Manager service to manage your network resources.",
```

Also the description seems vague to me.